modules/form-field/react/stories/stories_Checkbox.tsx

```diff
@@ -160,24 +160,26 @@ storiesOf('Components|Inputs/Checkbox/React', module)
                     return;
                   }
 
+                  const key = `${checked ? 'checked' : 'unchecked'}-${
```

Comment:
Having some problems with key errors in the story - can't figure out why. Getting `react.development.js:188 Warning: Each child in a list should have a unique "key" prop.`. However, I've programmatically checked all of the keys to ensure there are no duplicates. Anyone see something I'm missing?

Reply:
It looks like each children in the return needs to have a key. Something like this removes the error:
```
return (
  <tr key={`${key}-tr`}>
    <td key={`${key}-td`}>{`${disabled ? 'Disabled ' : ''}${variant} (${
      checked ? (indeterminate ? 'indeterminate' : 'checked') : 'unchecked'
      })`}</td>

    {['', 'hover', 'focus', 'active'].map(className => (
      <td
        key={`${key}-${className}-td`}>
        <Checkbox
          checked={checked}
          disabled={disabled}
          indeterminate={indeterminate}
          error={type}
          className={className}
          key={`${key}-${className}-checkbox`}
          onChange={() => { }} // eslint-disable-line no-empty-function
        />
      </td>
    ))}
  </tr>
);
```

Reply:
Interesting... I thought it was only the top level one for some reason.

Reply:
@stephanerangaya It is indeed only needed at the top level - I was just silly and had the `key` on the `Checkbox` instead of its containing `td`.

Reply:
Oh, I see!